### ISSUE TITLE
`pnpm run dev` throwing error

### ISSUE THREAD

**Issue by tedmoore** [flucoma/learn-website#177]
I tried to eyeball which `glob` I might be missing or whatever, but my eyes are not sensitive to the varieties of globs. Thanks for any guidance!

```
macprocomputer@macprocomputers-MacBook-Pro learn-website % pnpm run dev

> learn@0.0.1 dev /Users/macprocomputer/Desktop/_flucoma/code/learn-website
> node scripts/metadata.js && node scripts/db.js && svelte-kit dev --host

file:///Users/macprocomputer/Desktop/_flucoma/code/learn-website/scripts/git.js:10
	let lastCommit = commitData.split(' ')[1].trim().slice(0, 6);
	                                         ^

TypeError: Cannot read properties of undefined (reading 'trim')
    at parseGitLog (file:///Users/macprocomputer/Desktop/_flucoma/code/learn-website/scripts/git.js:10:43)
    at file:///Users/macprocomputer/Desktop/_flucoma/code/learn-website/scripts/git.js:21:34
    at Array.forEach (<anonymous>)
    at extractGit (file:///Users/macprocomputer/Desktop/_flucoma/code/learn-website/scripts/git.js:18:9)
    at file:///Users/macprocomputer/Desktop/_flucoma/code/learn-website/scripts/metadata.js:20:10
    at f (/Users/macprocomputer/Desktop/_flucoma/code/learn-website/node_modules/.pnpm/once@1.4.0/node_modules/once/once.js:25:25)
    at Glob.<anonymous> (/Users/macprocomputer/Desktop/_flucoma/code/learn-website/node_modules/.pnpm/glob@7.2.0/node_modules/glob/glob.js:148:7)
    at Glob.emit (node:events:520:28)
    at Glob._finish (/Users/macprocomputer/Desktop/_flucoma/code/learn-website/node_modules/.pnpm/glob@7.2.0/node_modules/glob/glob.js:194:8)
    at done (/Users/macprocomputer/Desktop/_flucoma/code/learn-website/node_modules/.pnpm/glob@7.2.0/node_modules/glob/glob.js:179:14)

Node.js v17.4.0
 ELIFECYCLE  Command failed with exit code 1.
macprocomputer@macprocomputers-MacBook-Pro learn-website % 
```

**Comment by jdchart** [comment]
I find this happens periodically. I tend to do one of two things which always seem to work:
- push a commit on your branch if you've just branched.
- reinstall with pnpm -i or whatever it is (sorry im not on my computer)

**Comment by tedmoore** [comment]
> * push a commit on your branch if you've just branched.

This did it! Thank you @jdchart !

**Comment by jamesb93** [comment]
Ah yes, part of the metadata scraping is to rip through the `git log` and find the last modification time, author and commit of a file. If there is no commits it barfs - I should fix this probably 😂

**Comment by jamesb93** [comment]
Fixed in #179
